tests/SprykerEcoTest/Zed/Braintree/Business/BraintreeFacadeCaptureTest.php

```diff
@@ -44,27 +47,87 @@ public function testCapturePaymentWithSuccessResponse()
     /**
      * @return void
      */
-    public function testCapturePaymentWithFailureResponse()
+    public function testCapturePaymentWithFailureResponse(): void
     {
-        $factoryMock = $this->getFactoryMock(['createCaptureTransaction']);
-        $factoryMock->method('createCaptureTransaction')->willReturn(
-            $this->getCaptureTransactionMock(false)
+        $factoryMock = $this->getFactoryMock(['createCaptureOrderTransaction']);
+        $factoryMock->method('createCaptureOrderTransaction')->willReturn(
+            $this->getCaptureOrderTransactionMock(false)
         );
         $braintreeFacade = $this->getBraintreeFacade($factoryMock);
         $response = $braintreeFacade->capturePayment($this->getTransactionMetaTransfer());
 
         $this->assertFalse($response->getIsSuccess());
     }
 
+    /**
+     * @return void
+     */
+    public function testCaptureOrderPaymentWithSuccessResponse(): void
+    {
+        $factoryMock = $this->getFactoryMock(['createCaptureOrderTransaction']);
```

Comment:
Not a blocker, but please, try do not forget about Arrange-Act-Assert structure. It makes tests more readable